includes/reader-activation/class-reader-activation.php

```diff
@@ -1739,8 +1739,9 @@ public static function canonize_user_data( $user_data = [] ) {
 		$user_data = array_merge(
```

Comment:
It doesn't let me make a comment higher up than this, but it looks like `$user_login` isn't used anywhere any more after this change, so should be removed. Not a blocker though.

Reply:
Huh, good catch! Fixed in 742c30a13477b99e6f721e889f7407e655333814